css/mobile.css

```
@@ -0,0 +1,256 @@
+    text-indent: -9999em;
+    margin: 88px auto;
+    position: relative;
+    font-size: 11px;
+    transform: translateZ(0);
+    animation-delay: -0.16s;
+}
+
+    ::before,
+    ::after {
+        position: absolute;
+        top: 0;
+        content: '';
+    }
+
+    ::before {
+        left: -1.5em;
+        animation-delay: -0.32s;
+    }
+
+    ::after {
+        left: 1.5em;
+    }
+
+
+
+
+@-webkit-keyframes loading {
+
+    0%,
+    80%,
+    100% {
+        box-shadow: 0 0;
+        height: 4em;
+    }
+
+    40% {
+        box-shadow: 0 -2em;
+        height: 5em;
+    }
+}
+
+@keyframes loading {
+
+    0%,
+    80%,
+    100% {
+        box-shadow: 0 0;
+        height: 4em;
+    }
+
+    40% {
+        box-shadow: 0 -2em;
+        height: 5em;
+    }
+}
+
+.loaded {
+    opacity: 0;
+    visibility: hidden;
+  }
+
+
+
+.header-parent{
+    background-color:#333;
+    position:fixed;
+    top:0;
+    left:0;
+    margin-bottom: 50px;
+    width:100%;
+    height:50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 10;
+}
+
+.header-name{
+    padding-left: 20px;
+    color:#F4F5F7;
+    font-family:Georgia, 'Times New Roman', Times, serif;
+    font-size:19px;
+}
+
+.header-instaicon{
+    text-decoration: none;
+    width:8%;
+    margin-right:15px;
+}
+
+.instaicon{
+transition: all 1s;
+width:100%;
+height:auto;
+filter: invert();
+}
+
+
+ .instaicon:hover{
+    transform:scale(1.3);
+}
+
+.titletop{
+    width:100%;
+    text-align: center;
+    font-family: 'Dancing Script', cursive;
+    font-size: 20px;
+}
+
+.name{
+    animation-delay:3s;
+    animation-name:fadeUpAnime;
+    animation-duration:3s;
+    animation-fill-mode:forwards;
+    opacity:0;
+    }
+    
+    @keyframes fadeUpAnime{
+      from {
+        opacity: 0;
+      transform: translateY(100px);
+      }
+    
+      to {
+        opacity: 1;
+      transform: translateY(0);
+      }
+    }
+
+    .photograph{
+        animation-delay:5s;
+        animation-name:fadeLeftAnime;
+        animation-duration:1s;
+        animation-fill-mode:forwards;
+        opacity:0;
+        }
+        
+        @keyframes fadeLeftAnime{
+          from {
+            opacity: 0;
+          transform: translateX(-100px);
+          }
+        
+          to {
+            opacity: 1;
+          transform: translateX(0);
+          }
+        }
+
+.photograph{
+    width:55%;
+    margin-top:20px;
+    height:auto;
+}
+
+.photograph2{
+    width:60%;
+    margin-top:20px;
+    height:auto;
+}
+
+.contents-parent{
+    display: flex;
+    justify-content: center;
+    margin-bottom: 50px;
+    text-align:center;
+    margin-top: 40px;
+    flex-wrap: wrap;
+
+}
+
+.contents-item{
+    width:40%;
+    background-color:#333;
+    margin:10px;
+    height:220px;
+    border-radius: 15px;
+    padding-bottom:20px;
+}
+.contents-item h2{
+    color: #F4F5F7;
+}
+
+.contents-item p{
+    color:#F4F5F7;
+    font-size:10px;
+}
```

伝搬するので、`color:#F4F5F7;`は`.contents-item`に入れてokです
```suggestion
.contents-item{
    width:40%;
    background-color:#333;
    margin:10px;
    height:220px;
    border-radius: 15px;
    padding-bottom:20px;
    color:#F4F5F7;
}

.contents-item p{
    font-size:10px;
}
```